receiver/awscontainerinsightreceiver/internal/efa/efaSysfs_test.go

```diff
@@ -4,8 +4,10 @@
 package efa
 
 import (
+	"cmp"
```

Comment:
It's complaining about this package because it was introduced in 1.21 and the go mod has a minimum of 1.20. Can use https://pkg.go.dev/github.com/google/go-cmp/cmp instead.

Reply:
That `cmp` didn't really have what I needed, so I just copied the one function I needed from the go 1.21 `cmp`.